packages/scripts/src/commands/app-data/postProcess/assets.ts

```diff
@@ -95,16 +97,26 @@ export class AssetsPostProcessor {
     assetEntries: IAssetEntryHashmap,
     missingEntries: IAssetEntryHashmap
   ) {
-    const contentsTarget = path.resolve(appAssetsFolder, "contents.json");
-    fs.writeFileSync(contentsTarget, JSON.stringify(sortJsonKeys(assetEntries), null, 2));
-    const missingTarget = path.resolve(appAssetsFolder, "untracked-assets.json");
-    if (fs.existsSync(missingTarget)) fs.removeSync(missingTarget);
-    if (Object.keys(missingEntries).length > 0) {
-      logWarning({
-        msg1: "Assets override found without corresponding entry",
-        msg2: Object.keys(missingEntries).join("\n"),
-      });
-      fs.writeFileSync(missingTarget, JSON.stringify(sortJsonKeys(missingEntries), null, 2));
+    if (fs.existsSync(appAssetsFolder)) {
+      {
```

Comment:
Code tidy: there's an additional set of curly brackets around this block that should be removed